torch/nn/qat/modules/linear.py

```diff
@@ -2,7 +2,7 @@
 import torch.nn.functional as F
 from torch.nn.intrinsic import LinearReLU
 
-class Linear(nn.Linear):
+class Linear(nn.Linear, _FusedModule):
```

Comment:
I feel like patterns such as this might cause a diamond problem if the first class is also inheriting from sequential

Reply:
right, but we probably won't have that I think